packages/components/src/internal/components/forms/BulkAddUpdateForm.tsx

```diff
@@ -32,13 +36,17 @@ export const BulkAddUpdateForm: FC<Props> = props => {
     }, [data, dataKeys, editorModel, queryInfo, selectedRowIndexes]);
 
     return (
-        <QueryInfoForm
-            {...queryInfoFormProps}
-            fieldValues={fieldValues}
-            queryInfo={queryInfo.getInsertQueryInfo()}
-            submitForEditText={submitForEditText}
-            title={title}
-        />
+        <>
+            {warning && <Alert bsStyle="warning">{warning}</Alert>}
```

Comment:
If you use the labkey version of `Alert`, this simplifies to 
```suggestion
            <Alert bsStyle="warning">{warning}</Alert>
```